osgeo_importer/views.py

```diff
@@ -1,20 +1,27 @@
 import os
 import json
 import logging
+import shutil
 from django.http import HttpResponse
 from django.views.generic import FormView, ListView, TemplateView
 from django.core.urlresolvers import reverse_lazy
 from .forms import UploadFileForm
-from .models import UploadedData, UploadLayer, DEFAULT_LAYER_CONFIGURATION
+from .models import UploadedData, UploadLayer, UploadFile, DEFAULT_LAYER_CONFIGURATION
 from .importers import OSGEO_IMPORTER
 from .inspectors import OSGEO_INSPECTOR
 from .utils import import_string
 from django.conf import settings
+from django.core.files.storage import FileSystemStorage
 
 OSGEO_INSPECTOR = import_string(OSGEO_INSPECTOR)
 OSGEO_IMPORTER = import_string(OSGEO_IMPORTER)
 
 logger = logging.getLogger(__name__)
+if logger.handlers:
+    for handler in logger.handlers:
+        logger.removeHandler(handler)
+logging.basicConfig(filename='/vagrant/views.log', level=logging.DEBUG)
```

Comment:
1. doing logger setup like calling logging.basicConfig at import time isn't too great in any case, but specifically undesirable for something intended to be a reusable component like a library or reusable django app. the reason is that individual users will have their individual logging setups, all we want to do is expose enough structure in the logging that they can e.g. turn down our debug spam without affecting other apps' logging

2. why are we removing all existing handlers here? even if we want this, leave a comment to explain why.

3. this is hardcoding a filename with /vagrant/ in it